SeqMaster.py

```diff
@@ -0,0 +1,104 @@
+from typing import List, Union
+import scripts.dna_rna_tools as nucl
+import scripts.protein_tools as prot
+import scripts.fastaq_filter as fasq
+
+
+def run_dna_rna_tools(*args: str, seq_type='DNA') -> Union[list[str], str, list[float], float]:
+    """
+    Launch desired operation with nucleic acid sequences. Pass comma-separated sequences,
+    additional argument (if certain function requires it) and specify function name you want to apply to all sequences.
+    Pass arguments strictly in this order, otherwise it won't be parsed.
+
+    :param args:
+    - nucleic acid sequences for analysis (str)
+    - operation name (str): specify procedure you want to apply (str)
+    :param seq_type: type of desired complement sequence (for complement function) (str)
+
+    :return: the result of procedure in list or str format
+    """
+    seqs = args[:-1]
+    tool = args[-1]
+    seqs_identity = nucl.nucl_acid_identity(seqs)
+    if tool == 'nucl_acid_identity':
+        return seqs_identity
+    elif tool == 'complement':
+        return nucl.complement(seqs, seq_type)
+    elif tool == 'reverse_complement':
+        return nucl.reverse_complement(seqs, seq_type)
+    elif tool == 'transcribe':
+        return nucl.transcribe(seqs, seqs_identity)
+    elif tool == 'reverse':
+        return nucl.reverse(seqs)
+    elif tool == 'gc_content':
+        return nucl.gc_content(seqs)
+    else:
+        raise ValueError(f'There is no {tool} tool available!')
+
+
+def run_protein_analysis(*args: str, site_of_interest=None) -> Union[List[str], str, list[float], float]:
+    """
+    Launch desired operation with proteins sequences. Pass comma-separated sequences,
+    additional argument (if certain function requires it) and specify function name you want to apply to all sequences.
+    Pass arguments strictly in this order, otherwise it won't be parsed.
+
+    :param args:
+    - seq (str): amino acids sequences for analysis in 1-letter or 3-letter code (as many as you wish)
+    - curr_encoding (int): type of encoding of given protein sequences
+    - operation name (str): specify procedure you want to apply
+    :param site_of_interest: one letter encoding of desired site (for find_site function)
+    :return: the result of procedure in list or str format
+    """
+    tool = args[-1]
+    curr_encoding = args[-2]
+    seqs = args[:-2]
```

Comment:
Можно сделать более короткую запись:
```suggestion
    *seqs, curr_encoding, tool = args
```